src/translators/crate.py

```diff
@@ -57,8 +57,11 @@ def setup(self):
         if self.connection is None:
             try:
                 # Added backoff_factor for retry interval between attempt of consecutive retries
-                self.connection = client.connect([url], error_trace=True, backoff_factor=0.1)
-                self.ccm.set_connection('crate', self.connection)
+                backoff_factor = 0.0
+                if os.environ.get("backoff_factor"):
```

Comment:
this is crate specific, so better call it `CRATE_BACKOFF_FACTOR`

Reply:
this should be added in the documentation in the file configuration.md

Reply:
@daminichopra the branch is now clean, if you fix my comments above (after synching your local branch with the new remote!!!), we can then merge

Reply:
> this is crate specific, so better call it `CRATE_BACKOFF_FACTOR`

@daminichopra I think you missed this comment

Reply:
> this is crate specific, so better call it `CRATE_BACKOFF_FACTOR`

@chicco785 , backoff_factor is used in crate [code](https://github.com/crate/crate-python/blob/4bf932ace34fd4963f26bccccbedc4ac74ab3282/src/crate/client/http.py#L330)  which is being used in crate.py.
If i used `crate_backoff_factor` it will not recognize this keyword 
Also, let me knowif i missed something. Thanks

Reply:
sure, but this is unrelated to the name you give to the ENV VARIABLE :)